src/resources/index.html

```diff
@@ -0,0 +1,169 @@
+          <p class="mb-4">Aside from the palette itself, here are some useful web assets</p>
+          <ul class="list-disc list-inside">
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/tailwind">
+                Tailwind config
+              </a>
+            </li>
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/tridactyl">Tridactyl theme</a>
+              <span class="text-sm text-white">- This will be moved to a new repo soon</span>
+            </li>
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/userstyles">User styles</a>
+              <span class="text-sm text-white">- Using stylish extension for example</span>
+            </li>
+
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-web/tree/master/palette">Palette</a>
+              <span class="text-sm text-white">this is deprected for the palette above, kept for compatibility</span>
+            </li>
+          </ul>
+        </div>
+        <div class="mb-8">
+          <h2 class="text-xl mb-4 text-center sm:text-left uppercase">Graphic assets</h2>
+          <p class="mb-4">
+            We keep graphic assets in a separate repo so not to inflate the
+            size of each theme. We also keep the source svgs for various
+            assets, kept in the <code class="inline-code">src</code> folder.
+          </p>
+          <p class="mb-4">
+            If you are looking for icons, logo, screenshots etc this is the place.
+          </p>
+          <ul class="list-disc list-inside">
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-assets">
+                Github repository
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="mb-8">
+          <h2 class="text-xl mb-4 text-center sm:text-left uppercase">Miscellaneous</h2>
+          <p class="mb-4">
+            Here are some miscellaneous projects that don't really fit anywhere
+            else. Mostly experiments and and other bits and bobs.
+          </p>
+          <ul class="list-disc list-inside">
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-convert">Srcery Website</a>
+              <span class="text-sm text-white">- Source code for this site</span>
+            </li>
+            <li class="mb-2">
+              <a target="_blank" class="link" href="https://github.com/srcery-colors/srcery-convert">Srcery Convert</a>
+              <span class="text-sm text-white">- Rust program to convert an image's colors to srcery colors, your milage may vary</span>
```

Comment:
```suggestion
              <span class="text-sm text-white">- Rust program to convert an images' colors to srcery colors, your milage may vary</span>
```

Might need research.

Reply:
Rephrased it slightly, to reduce awkwardness ;)

Reply:
[bd52e8](https://github.com/srcery-colors/srcery-colors.github.io/pull/440/commits/3bd52e89de10a1560556e61521d66baf3742b21b)

Reply:
ok, so not super happy with that, gonna force push another try